cmake/jana_plugin.cmake

```diff
@@ -345,7 +345,7 @@ endmacro()
 macro(plugin_add_event_model _name)
 
   if(NOT podio_FOUND)
-    find_package(podio ${podio_VERSION_MIN})
+    find_package(podio ${podio_VERSION_MIN} QUIET)
     if(NOT podio_FOUND)
       find_package(podio 1.0 REQUIRED)
```

Comment:
Version from variable?

Reply:
Could too. I just copied the approach we have in edm4eic for consistency without our software stack.

Reply:
Though I see the argument that the consistency could also be with what we do for Acts.

Reply:
Oh, nevermind. I get it. "1.0" is not driven by a minimal version, even though the code would need to be updated once `podio_VERSION_MIN` goes up to 1.